app/pages/reservation/reservation-time/ReservationTime.js

```diff
@@ -42,9 +43,13 @@ class ReservationTime extends Component {
       selectedReservation,
       t,
       unit,
-      isStaff
+      isStaff,
+      date
     } = this.props;
-    const date = moment(selectedReservation.begin).format('YYYY-MM-DD');
+
```

Comment:
Reservation edit page take date from editted reservation, no need for those passing, which will confuse the component